control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1279,10 +1279,26 @@ func (r *HostedControlPlaneReconciler) reconcilePKI(ctx context.Context, hcp *hy
 		return fmt.Errorf("failed to reconcile kas kubelet client secret: %w", err)
 	}
 
-	// KAS aggregator cert secret
+	// KAS aggregator client signer
+	kasAggregateClientSigner := manifests.AggregateClientSigner(hcp.Namespace)
+	if _, err := createOrUpdate(ctx, r, kasAggregateClientSigner, func() error {
+		return pki.ReconcileAggregateClientSigner(kasAggregateClientSigner, p.OwnerRef)
+	}); err != nil {
+		return fmt.Errorf("failed to reconcile root CA: %w", err)
+	}
+
+	// KAS aggregator client CA
+	kasAggregateClientCA := manifests.AggregateClientCAConfigMap(hcp.Namespace)
+	if _, err := createOrUpdate(ctx, r, kasAggregateClientCA, func() error {
+		return pki.ReconcileAggregateClientCA(kasAggregateClientCA, p.OwnerRef, kasAggregateClientSigner)
+	}); err != nil {
+		return fmt.Errorf("failed to reconcile combined CA: %w", err)
+	}
```

Comment:
It seems that the configmap is missing in the control-plane of the aws run?

Also, naming should be:
```suggestion
	kasAggregatorClientCA := manifests.AggregatorClientCAConfigMap(hcp.Namespace)
	if _, err := createOrUpdate(ctx, r, kasAggregatorClientCA, func() error {
		return pki.ReconcileAggregatorClientCA(kasAggregatorClientCA, p.OwnerRef, kasAggregatorClientSigner)
	}); err != nil {
		return fmt.Errorf("failed to reconcile combined CA: %w", err)
	}
```
This is not an aggregate of anything but the client cert of the KAS API aggregator.